swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/Dates.kt

```diff
@@ -50,6 +52,12 @@ class Dates {
 
       throw exception!!
     }
+
+    fun toCreationDate(timestampMillis: Long) : String {
```

Comment:
Since this function is not specific to creation dates, it makes sense to rename it to something more generic for dates.

Reply:
It takes a timestamp and changes it to the 'aws creation date' format. I think the name is fine, given that that is the functionality that it is doing.